CONTRIBUTING.md

```diff
@@ -42,6 +42,8 @@ Pull requests (PRs)
 
 - To avoid duplicate work, create a `[WIP]` pull request as soon as possible.
 - Your PR must include **test coverage.** See [test/README.md][run-tests].
+- Runtimes fixes should follow [Vim conventions][vim-contribute-runtime] first,
```

Comment:
This means basically nothing to new contributors. And the "and backported" blurb is also misleading, it implies the contributor is the one who should backport it.